composer.json

```diff
@@ -22,16 +22,18 @@
     },
     "require": {
         "php": "^7.2",
+        "ext-ast": "*",
```

Comment:
This should be under require-dev, no? Also need to run `composer update` to make the lock file in sync